TESTS/mbed_hal/flash/functional_tests/main.cpp

```diff
@@ -266,20 +266,20 @@ void flash_buffer_alignment_test()
 
 // check the execution speed at the start and end of the test to make sure
 // cache settings weren't changed
-void flash_clock_and_cache_test()
-{
-    const int timer_diff_end = time_cpu_cycles(TEST_CYCLES);
-    const int acceptable_range = timer_diff_start / (ALLOWED_DRIFT_PPM);
-    TEST_ASSERT_UINT32_WITHIN(acceptable_range, timer_diff_start, timer_diff_end);
-}
+//void flash_clock_and_cache_test()
```

Comment:
can we just comment call to this function.

Reply:
I didn't want to throw an extra warning about an unused function.